source/_patterns/01-molecules/components/suggestions.twig

```diff
@@ -0,0 +1,7 @@
+<ul class="o-suggestions u-border--bottom">
```

Comment:
Add the following class: `u-theme--color--base`

Reply:
Done!